miniupnpc/portlistingparse.c

```diff
@@ -31,7 +31,7 @@ static const struct {
 
 /* Helper function */
 static UNSIGNED_INTEGER
-atoui(const char * p, int l)
+atoui2(const char * p, int l)
```

Comment:
There is no atoui function in BeOS and there is no atoui function in any standard or extension I can find. So I think we should instead delete that function from Haiku?

Reply:
I think it would be best to just remove this function from Haiku. I can't find any software that uses the function directly.

Reply:
even if you choose to keep atoui2() in haiku you shouldn't modify miniupnpc that way.
just add
```
#if defined(__HAIKU__)
#define atoui atoui2
#endif 
```
at the top of the file...

Reply:
The version in Haiku has a different prototype (unsigned atoui(const char*) so that wouldn't work

Reply:
what do you mean ?
`#define atoui atoui2` at the top of the fie (but after the system #includes) is just like replacing every occurence of atoui by atoui2 in the remaining of the source file

Reply:
I'll give this a go when I have time.